chapter_1.py

```diff
@@ -1,26 +1,23 @@
 print("One late Thursday morning, as they gathered at the haunted pizza place,")
-print ("there was a skull sitting on the counter, humming.")
+print ("There was a skull sitting on the counter, humming.")
```

Comment:
you still need a lowercase T because the previous line has a comma not a period.